### ISSUE TITLE
Error: The binding type(s) 'queueTrigger, queue' were not found in the configured extension bundle. Please ensure the type is correct and the correct version of extension bundle is configured.

### ISSUE THREAD

**Issue by rajeshr6r** [Azure/azure-functions-host#6359]
Hello,

I have started receiving this error just today . I thought I might have an issue and updated the Core Tools and currently running the following . 

Azure Functions Core Tools (2.7.2628 Commit hash: 42e149f5313330f1f0a5e721a7e59d9ef7d27e6f)
Function Runtime Version: 2.0.13907.0
Azure VS Code Functions Extension : v0.23.0
Language : Python 
Azure Function Type : Queue Trigger 

The binding type(s) 'queueTrigger, queue' were not found in the configured extension bundle. Please ensure the type is correct and the correct version of extension bundle is configured.

Please can this be checked ?

**Comment by rajeshr6r** [comment]
Thought of adding the log to this 

Azure Functions Core Tools (2.7.2628 Commit hash: 42e149f5313330f1f0a5e721a7e59d9ef7d27e6f)
Function Runtime Version: 2.0.13907.0
AZURE_FUNCTIONS_ENVIRONMENT: Development
[16-07-2020 04:28:16] Building host: startup suppressed: 'False', configuration suppressed: 'False', startup operation id: '525e3cec-a965-4cd1-8201-88b400a8af5f'
[16-07-2020 04:28:17] Reading host configuration file 'D:\Azure\AzureFunctions\OutboundService\host.json'
[16-07-2020 04:28:17] Host configuration file read:
[16-07-2020 04:28:17] {
[16-07-2020 04:28:17]   "version": "2.0",
[16-07-2020 04:28:17]   "extensionBundle": {
[16-07-2020 04:28:17]     "id": "Microsoft.Azure.Functions.ExtensionBundle",
[16-07-2020 04:28:17]     "version": "[1.*, 2.0.0)"
[16-07-2020 04:28:17]   }
[16-07-2020 04:28:17] }
[16-07-2020 04:28:18] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:java
[16-07-2020 04:28:18] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:node
[16-07-2020 04:28:18] File 'C:\Program Files\dotnet\dotnet.exe' is not found, 'dotnet' invocation will rely on the PATH environment variable.
[16-07-2020 04:28:18] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:powershell
[16-07-2020 04:28:18] Loading functions metadata
[16-07-2020 04:28:18] Reading functions metadata
[16-07-2020 04:28:18] 3 functions found
[16-07-2020 04:28:18] 3 functions loaded
[16-07-2020 04:28:18] Looking for extension bundle Microsoft.Azure.Functions.ExtensionBundle at C:\Users\Vidhya Rajesh\AppData\Local\Temp\Functions\ExtensionBundles\Microsoft.Azure.Functions.ExtensionBundle
[16-07-2020 04:28:18] Found a matching extension bundle at C:\Users\Vidhya Rajesh\AppData\Local\Temp\Functions\ExtensionBundles\Microsoft.Azure.Functions.ExtensionBundle\1.3.0
[16-07-2020 04:28:18] Fetching information on versions of extension bundle Microsoft.Azure.Functions.ExtensionBundle available on https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/index.json
[16-07-2020 04:28:19] Skipping bundle download since it already exists at path C:\Users\Vidhya Rajesh\AppData\Local\Temp\Functions\ExtensionBundles\Microsoft.Azure.Functions.ExtensionBundle\1.3.0
[16-07-2020 04:28:19] Loading extension bundle from C:\Users\Vidhya Rajesh\AppData\Local\Temp\Functions\ExtensionBundles\Microsoft.Azure.Functions.ExtensionBundle\1.3.0
[16-07-2020 04:28:19] Loading startup extension 'AzureStorage'
[16-07-2020 04:28:19] Unable to load startup extension 'AzureStorage' (Type: 'Microsoft.Azure.WebJobs.Extensions.Storage.AzureStorageWebJobsStartup, Microsoft.Azure.WebJobs.Extensions.Storage, Version=3.0.11.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'). The type does not exist. Please validate the type and assembly names.
[16-07-2020 04:28:20] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:java
[16-07-2020 04:28:20] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:node
[16-07-2020 04:28:20] File 'C:\Program Files\dotnet\dotnet.exe' is not found, 'dotnet' invocation will rely on the PATH environment variable.
[16-07-2020 04:28:20] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:powershell
[16-07-2020 04:28:21] Initializing Warmup Extension.
[16-07-2020 04:28:21] Initializing Host. OperationId: '525e3cec-a965-4cd1-8201-88b400a8af5f'.
[16-07-2020 04:28:21] Host initialization: ConsecutiveErrors=0, StartupCount=1, OperationId=525e3cec-a965-4cd1-8201-88b400a8af5f[16-07-2020 04:28:21] LoggerFilterOptions
[16-07-2020 04:28:21] {
[16-07-2020 04:28:21]   "MinLevel": "None",
[16-07-2020 04:28:21]   "Rules": [
[16-07-2020 04:28:21]     {
[16-07-2020 04:28:21]       "ProviderName": null,
[16-07-2020 04:28:21]       "CategoryName": null,
[16-07-2020 04:28:21]       "LogLevel": null,
[16-07-2020 04:28:21]       "Filter": "<AddFilter>b__0"
[16-07-2020 04:28:21]     },
[16-07-2020 04:28:21]     {
[16-07-2020 04:28:21]       "ProviderName": "Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics.SystemLoggerProvider",
[16-07-2020 04:28:21]       "CategoryName": null,
[16-07-2020 04:28:21]       "LogLevel": "None",
[16-07-2020 04:28:21]       "Filter": null
[16-07-2020 04:28:21]     },
[16-07-2020 04:28:21]     {
[16-07-2020 04:28:21]       "ProviderName": "Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics.SystemLoggerProvider",
[16-07-2020 04:28:21]       "CategoryName": null,
[16-07-2020 04:28:21]       "LogLevel": null,
[16-07-2020 04:28:21]       "Filter": "<AddFilter>b__0"
[16-07-2020 04:28:21]     }
[16-07-2020 04:28:21]   ]
[16-07-2020 04:28:21] }
[16-07-2020 04:28:21] FunctionResultAggregatorOptions
[16-07-2020 04:28:21] {
[16-07-2020 04:28:21]   "BatchSize": 1000,
[16-07-2020 04:28:21]   "FlushTimeout": "00:00:30",
[16-07-2020 04:28:21]   "IsEnabled": true
[16-07-2020 04:28:21] }
[16-07-2020 04:28:21] SingletonOptions
[16-07-2020 04:28:21] {
[16-07-2020 04:28:21]   "LockPeriod": "00:00:15",
[16-07-2020 04:28:21]   "ListenerLockPeriod": "00:00:15",
[16-07-2020 04:28:21]   "LockAcquisitionTimeout": "10675199.02:48:05.4775807",
[16-07-2020 04:28:21]   "LockAcquisitionPollingInterval": "00:00:05",
[16-07-2020 04:28:21]   "ListenerLockRecoveryPollingInterval": "00:01:00"
[16-07-2020 04:28:21] }
[16-07-2020 04:28:21] HttpOptions
[16-07-2020 04:28:21] {
[16-07-2020 04:28:21]   "DynamicThrottlesEnabled": false,
[16-07-2020 04:28:21]   "MaxConcurrentRequests": -1,
[16-07-2020 04:28:21]   "MaxOutstandingRequests": -1,
[16-07-2020 04:28:21]   "RoutePrefix": "api"
[16-07-2020 04:28:21] }
[16-07-2020 04:28:21] Starting JobHost
[16-07-2020 04:28:21] Starting Host (HostId=laptophh58ibd5-1129759262, InstanceId=9ef2b861-4012-42aa-9a89-28614a05b217, Version=2.0.13907.0, ProcessId=4052, AppDomainId=1, InDebugMode=False, InDiagnosticMode=False, FunctionsExtensionVersion=(null))        
[16-07-2020 04:28:21] Loading functions metadata
[16-07-2020 04:28:21] Loading proxies metadata
[16-07-2020 04:28:22] Initializing Azure Function proxies
[16-07-2020 04:28:26] 0 proxies loaded
[16-07-2020 04:28:26] 3 functions loaded
[16-07-2020 04:28:26] Starting worker process:py  "c:/Users/Vidhya Rajesh/.vscode/extensions/ms-python.python-2020.6.91350/pythonFiles/lib/python/debugpy" --listen 127.0.0.1:9091 --wait-for-client "C:\Users\Vidhya Rajesh\AppData\Roaming\npm\node_modules\azure-functions-core-tools\bin\workers\python\3.6/WINDOWS/X64/worker.py" --host 127.0.0.1 --port 51159 --workerId a2368f6f-9b2b-4a53-b807-3c8f879125bb --requestId d69b788c-1349-4e62-bd60-0c89e41d825a --grpcMaxMessageLength 2147483647
[16-07-2020 04:28:26] py process with Id=20772 started
[16-07-2020 04:28:26] Generating 1 job function(s)
[16-07-2020 04:28:26] Found the following functions:
[16-07-2020 04:28:26] Host.Functions.StatusCallBackHTTP
[16-07-2020 04:28:26]
[16-07-2020 04:28:26] Function 'Functions.StatusCallBackHTTP' is disabled
[16-07-2020 04:28:26] Initializing function HTTP routes
[16-07-2020 04:28:26] Mapped function route 'api/StatusCallBackHTTP' [post] to 'StatusCallBackHTTP'
[16-07-2020 04:28:26]
[16-07-2020 04:28:26] Host initialized (5217ms)
[16-07-2020 04:28:26] Host started (5232ms)
[16-07-2020 04:28:26] Job host started
[16-07-2020 04:28:26] The 'CallBatchProcessing' function is in error: The binding type(s) 'queueTrigger, queue' were not found in the configured extension bundle. Please ensure the type is correct and the correct version of extension bundle is configured.
[16-07-2020 04:28:26] The 'CallQueueTrigger' function is in error: The binding type(s) 'queueTrigger, queue' were not found in the configured extension bundle. Please ensure the type is correct and the correct version of extension bundle is configured.
Hosting environment: Development
Content root path: D:\Azure\AzureFunctions\OutboundService\OutboundService
Now listening on: http://0.0.0.0:7071
Application started. Press Ctrl+C to shut down.
Function StatusCallBackHTTP is disabled.
[16-07-2020 04:28:34] Host lock lease acquired by instance ID '00000000000000000000000029167309'.
[16-07-2020 04:28:38]  INFO: Starting Azure Functions Python Worker.
[16-07-2020 04:28:38]  INFO: Worker ID: a2368f6f-9b2b-4a53-b807-3c8f879125bb, Request ID: d69b788c-1349-4e62-bd60-0c89e41d825a, Host Address: 127.0.0.1:51159
[16-07-2020 04:28:38]  INFO: Successfully opened gRPC channel to 127.0.0.1:51159
[16-07-2020 04:28:38]  INFO: Detach console logging. Switch to gRPC logging
[16-07-2020 04:28:38] Detach console logging. Switch to gRPC logging
[16-07-2020 04:28:38]  INFO: Received WorkerInitRequest, request ID d69b788c-1349-4e62-bd60-0c89e41d825a
[16-07-2020 04:28:38] Received WorkerInitRequest, request ID d69b788c-1349-4e62-bd60-0c89e41d825a
[16-07-2020 04:28:38]  INFO: Received FunctionLoadRequest, request ID: d69b788c-1349-4e62-bd60-0c89e41d825a, function ID: c6600bb2-e894-4218-b10f-3086714449eb
[16-07-2020 04:28:38] Received FunctionLoadRequest, request ID: d69b788c-1349-4e62-bd60-0c89e41d825a, function ID: c6600bb2-e894-4218-b10f-3086714449eb
[16-07-2020 04:28:48]  INFO: Successfully processed FunctionLoadRequest, request ID: d69b788c-1349-4e62-bd60-0c89e41d825a, function ID: c6600bb2-e894-4218-b10f-3086714449eb
[16-07-2020 04:28:48] Successfully processed FunctionLoadRequest, request ID: d69b788c-1349-4e62-bd60-0c89e41d825a, function ID: c6600bb2-e894-4218-b10f-3086714449eb

**Comment by rajeshr6r** [comment]
I tried deleting the extension bundle and re-started debugging which forced the bundle to be downloaded again from the Azure Functions CDN and suddenly this error vanished . 

I observed that the bundles were last updated on 06-07-2020 ( as per the folder modified date that I saw ) .

If you observe after the Job Host Started step , the error is not there anymore . I'll try running debugging locally and confirm.




Azure Functions Core Tools (2.7.2628 Commit hash: 42e149f5313330f1f0a5e721a7e59d9ef7d27e6f)
Function Runtime Version: 2.0.13907.0
AZURE_FUNCTIONS_ENVIRONMENT: Development
[16-07-2020 05:02:28] Building host: startup suppressed: 'False', configuration suppressed: 'False', startup operation id: '3ed06f49-0e81-41ef-b962-13bb9dd43f3c'
[16-07-2020 05:02:28] Reading host configuration file 'D:\Azure\AzureFunctions\OutboundService\OutboundService\host.json'
[16-07-2020 05:02:28] Host configuration file read:
[16-07-2020 05:02:28] {
[16-07-2020 05:02:28]   "version": "2.0",
[16-07-2020 05:02:28]   "extensionBundle": {
[16-07-2020 05:02:28]     "id": "Microsoft.Azure.Functions.ExtensionBundle",
[16-07-2020 05:02:28]     "version": "[1.*, 2.0.0)"
[16-07-2020 05:02:28]   }
[16-07-2020 05:02:28] }
[16-07-2020 05:02:28] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:java
[16-07-2020 05:02:28] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:node
[16-07-2020 05:02:28] File 'C:\Program Files\dotnet\dotnet.exe' is not found, 'dotnet' invocation will rely on the PATH environment variable.
[16-07-2020 05:02:28] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:powershell
[16-07-2020 05:02:28] Loading functions metadata
[16-07-2020 05:02:28] Reading functions metadata
[16-07-2020 05:02:28] 3 functions found
[16-07-2020 05:02:28] 3 functions loaded
[16-07-2020 05:02:28] Looking for extension bundle Microsoft.Azure.Functions.ExtensionBundle at C:\Users\Vidhya Rajesh\AppData\Local\Temp\Functions\ExtensionBundles\Microsoft.Azure.Functions.ExtensionBundle
[16-07-2020 05:02:28] Fetching information on versions of extension bundle Microsoft.Azure.Functions.ExtensionBundle available on https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/index.json
[16-07-2020 05:02:30] Downloading extension bundle from https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.3.0/Microsoft.Azure.Functions.ExtensionBundle.1.3.0.zip to C:\Users\Vidhya Rajesh\AppData\Local\Temp\7872eb4a-48ba-43c1-9f35-d99bc3b03cd5\Microsoft.Azure.Functions.ExtensionBundle.1.3.0.zip
[16-07-2020 05:02:32] Completed downloading extension bundle from https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.3.0/Microsoft.Azure.Functions.ExtensionBundle.1.3.0.zip to C:\Users\Vidhya Rajesh\AppData\Local\Temp\7872eb4a-48ba-43c1-9f35-d99bc3b03cd5\Microsoft.Azure.Functions.ExtensionBundle.1.3.0.zip
[16-07-2020 05:02:32] Extracting extension bundle at C:\Users\Vidhya Rajesh\AppData\Local\Temp\Functions\ExtensionBundles\Microsoft.Azure.Functions.ExtensionBundle\1.3.0
[16-07-2020 05:02:34] Zip extraction complete
[16-07-2020 05:02:34] Loading extension bundle from C:\Users\Vidhya Rajesh\AppData\Local\Temp\Functions\ExtensionBundles\Microsoft.Azure.Functions.ExtensionBundle\1.3.0
[16-07-2020 05:02:34] Loading startup extension 'AzureStorage'
[16-07-2020 05:02:35] Loaded extension 'AzureStorage' (3.0.11.0)
[16-07-2020 05:02:35] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:java
[16-07-2020 05:02:35] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:node
[16-07-2020 05:02:35] File 'C:\Program Files\dotnet\dotnet.exe' is not found, 'dotnet' invocation will rely on the PATH environment variable.
[16-07-2020 05:02:35] FUNCTIONS_WORKER_RUNTIME set to python. Skipping WorkerConfig for language:powershell
[16-07-2020 05:02:35] Initializing Warmup Extension.
[16-07-2020 05:02:35] Initializing Host. OperationId: '3ed06f49-0e81-41ef-b962-13bb9dd43f3c'.
[16-07-2020 05:02:36] Host initialization: ConsecutiveErrors=0, StartupCount=1, OperationId=3ed06f49-0e81-41ef-b962-13bb9dd43f3c
[16-07-2020 05:02:36] LoggerFilterOptions
[16-07-2020 05:02:36] {
[16-07-2020 05:02:36]   "MinLevel": "None",
[16-07-2020 05:02:36]   "Rules": [
[16-07-2020 05:02:36]     {
[16-07-2020 05:02:36]       "ProviderName": null,
[16-07-2020 05:02:36]       "CategoryName": null,
[16-07-2020 05:02:36]       "LogLevel": null,
[16-07-2020 05:02:36]       "Filter": "<AddFilter>b__0"
[16-07-2020 05:02:36]     },
[16-07-2020 05:02:36]     {
[16-07-2020 05:02:36]       "ProviderName": "Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics.SystemLoggerProvider",
[16-07-2020 05:02:36]       "CategoryName": null,
[16-07-2020 05:02:36]       "LogLevel": "None",
[16-07-2020 05:02:36]       "Filter": null
[16-07-2020 05:02:36]     },
[16-07-2020 05:02:36]     {
[16-07-2020 05:02:36]       "ProviderName": "Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics.SystemLoggerProvider",
[16-07-2020 05:02:36]       "CategoryName": null,
[16-07-2020 05:02:36]       "LogLevel": null,
[16-07-2020 05:02:36]       "Filter": "<AddFilter>b__0"
[16-07-2020 05:02:36]     }
[16-07-2020 05:02:36]   ]
[16-07-2020 05:02:36] }
[16-07-2020 05:02:36] FunctionResultAggregatorOptions
[16-07-2020 05:02:36] {
[16-07-2020 05:02:36]   "BatchSize": 1000,
[16-07-2020 05:02:36]   "FlushTimeout": "00:00:30",
[16-07-2020 05:02:36]   "IsEnabled": true
[16-07-2020 05:02:36] }
[16-07-2020 05:02:36] SingletonOptions
[16-07-2020 05:02:36] {
[16-07-2020 05:02:36]   "LockPeriod": "00:00:15",
[16-07-2020 05:02:36]   "ListenerLockPeriod": "00:00:15",
[16-07-2020 05:02:36]   "LockAcquisitionTimeout": "10675199.02:48:05.4775807",
[16-07-2020 05:02:36]   "LockAcquisitionPollingInterval": "00:00:05",
[16-07-2020 05:02:36]   "ListenerLockRecoveryPollingInterval": "00:01:00"
[16-07-2020 05:02:36] }
[16-07-2020 05:02:36] QueuesOptions
[16-07-2020 05:02:36] {
[16-07-2020 05:02:36]   "BatchSize": 16,
[16-07-2020 05:02:36]   "NewBatchThreshold": 64,
[16-07-2020 05:02:36]   "MaxPollingInterval": "00:00:02",
[16-07-2020 05:02:36]   "MaxDequeueCount": 5,
[16-07-2020 05:02:36]   "VisibilityTimeout": "00:00:00"
[16-07-2020 05:02:36] }
[16-07-2020 05:02:36] BlobsOptions
[16-07-2020 05:02:36] {
[16-07-2020 05:02:36]   "CentralizedPoisonQueue": false
[16-07-2020 05:02:36] }
[16-07-2020 05:02:36] HttpOptions
[16-07-2020 05:02:36] {
[16-07-2020 05:02:36]   "DynamicThrottlesEnabled": false,
[16-07-2020 05:02:36]   "MaxConcurrentRequests": -1,
[16-07-2020 05:02:36]   "MaxOutstandingRequests": -1,
[16-07-2020 05:02:36]   "RoutePrefix": "api"
[16-07-2020 05:02:36] }
[16-07-2020 05:02:36] Starting JobHost
[16-07-2020 05:02:36] Starting Host (HostId=laptophh58ibd5-1129759262, InstanceId=0c94e3f7-319e-49b0-b229-98d239a2552e, Version=2.0.13907.0, ProcessId=12792, AppDomainId=1, InDebugMode=False, InDiagnosticMode=False, FunctionsExtensionVersion=(null))
[16-07-2020 05:02:36] Loading functions metadata
[16-07-2020 05:02:36] Loading proxies metadata
[16-07-2020 05:02:36] Initializing Azure Function proxies
[16-07-2020 05:02:37] 0 proxies loaded
[16-07-2020 05:02:37] 3 functions loaded
[16-07-2020 05:02:37] Starting worker process:py  "c:/Users/Vidhya Rajesh/.vscode/extensions/ms-python.python-2020.6.91350/pythonFiles/lib/python/debugpy" --listen 127.0.0.1:9091 --wait-for-client "C:\Users\Vidhya Rajesh\AppData\Roaming\npm\node_modules\azure-functions-core-tools\bin\workers\python\3.6/WINDOWS/X64/worker.py" --host 127.0.0.1 --port 51460 --workerId bf2896bb-8e3a-42d4-b341-254229e92226 --requestId 8cfb1a9c-37f9-4749-9187-4db94ada5f6b --grpcMaxMessageLength 2147483647
[16-07-2020 05:02:37] py process with Id=6780 started
[16-07-2020 05:02:37] Generating 3 job function(s)
[16-07-2020 05:02:37] Found the following functions:
[16-07-2020 05:02:37] Host.Functions.CallBatchProcessing
[16-07-2020 05:02:37] Host.Functions.CallQueueTrigger
[16-07-2020 05:02:37] Host.Functions.StatusCallBackHTTP
[16-07-2020 05:02:37]
[16-07-2020 05:02:37] Function 'Functions.CallQueueTrigger' is disabled
[16-07-2020 05:02:37] Function 'Functions.StatusCallBackHTTP' is disabled
[16-07-2020 05:02:37] Initializing function HTTP routes
[16-07-2020 05:02:37] Mapped function route 'api/StatusCallBackHTTP' [post] to 'StatusCallBackHTTP'
[16-07-2020 05:02:37]
[16-07-2020 05:02:37] Host initialized (1103ms)
[16-07-2020 05:02:37] Host started (1160ms)
[16-07-2020 05:02:37] Job host started
Hosting environment: Development
Content root path: D:\Azure\AzureFunctions\OutboundService\OutboundService
Now listening on: http://0.0.0.0:7071
Application started. Press Ctrl+C to shut down.
Function CallQueueTrigger is disabled.
Function StatusCallBackHTTP is disabled.
[16-07-2020 05:02:44] Host lock lease acquired by instance ID '00000000000000000000000029167309'.
[16-07-2020 05:02:45]  INFO: Starting Azure Functions Python Worker.
[16-07-2020 05:02:45]  INFO: Worker ID: bf2896bb-8e3a-42d4-b341-254229e92226, Request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, Host Address: 127.0.0.1:51460
[16-07-2020 05:02:45]  INFO: Successfully opened gRPC channel to 127.0.0.1:51460
[16-07-2020 05:02:45]  INFO: Detach console logging. Switch to gRPC logging
[16-07-2020 05:02:45] Detach console logging. Switch to gRPC logging
[16-07-2020 05:02:45]  INFO: Received WorkerInitRequest, request ID 8cfb1a9c-37f9-4749-9187-4db94ada5f6b
[16-07-2020 05:02:45] Received WorkerInitRequest, request ID 8cfb1a9c-37f9-4749-9187-4db94ada5f6b
[16-07-2020 05:02:45]  INFO: Received FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: d4911726-a2bf-4e43-a364-78e47eec8061
[16-07-2020 05:02:45] Received FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: d4911726-a2bf-4e43-a364-78e47eec8061[16-07-2020 05:02:47]  INFO: Successfully processed FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: d4911726-a2bf-4e43-a364-78e47eec8061
[16-07-2020 05:02:47] Successfully processed FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: d4911726-a2bf-4e43-a364-78e47eec8061
[16-07-2020 05:02:47]  INFO: Received FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: c32b9e0d-4155-4fdf-afb3-f1a7b49083e8
[16-07-2020 05:02:47] Received FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: c32b9e0d-4155-4fdf-afb3-f1a7b49083e8[16-07-2020 05:02:47]  INFO: Successfully processed FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: c32b9e0d-4155-4fdf-afb3-f1a7b49083e8
[16-07-2020 05:02:47]  INFO: Received FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: 93e6f7a7-f358-4976-a971-189a66dabc99
[16-07-2020 05:02:47] Successfully processed FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: c32b9e0d-4155-4fdf-afb3-f1a7b49083e8
[16-07-2020 05:02:48]  INFO: Successfully processed FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: 93e6f7a7-f358-4976-a971-189a66dabc99
[16-07-2020 05:02:48] Received FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: 93e6f7a7-f358-4976-a971-189a66dabc99[16-07-2020 05:02:48] Successfully processed FunctionLoadRequest, request ID: 8cfb1a9c-37f9-4749-9187-4db94ada5f6b, function ID: 93e6f7a7-f358-4976-a971-189a66dabc99
